docs/writing-policies/rust/04-write-validation-logic.md

```diff
@@ -87,16 +85,15 @@ fn validate(payload: &[u8]) -> CallResult {
 
 ## Unit tests
 
-Finally, we will create some unit tests to ensure the validation code works as
+Now, you need to create unit tests to check the validation code works as
 expected.
 
-The `lib.rs` file has already some tests defined at the bottom of the file, as
-you can see Kubewarden's Rust SDK provides some test helpers too.
+The `lib.rs` file already tests defined at the bottom of the file and, as
+you can see, Kubewarden's Rust SDK provides test helpers too.
 
-Moreover, the scaffolded project already ships with some default
-[test fixtures](https://en.wikipedia.org/wiki/Test_fixture#Software) inside of
-the `test_data` directory. We are going to take advantage of these recorded
-admission requests to write our unit tests.
+Moreover, the scaffolding ships with default [test fixtures](https://en.wikipedia.org/wiki/Test_fixture#Software) in the `test_data` directory.
+<!--TODO: Is that the right word, 'recorded'?-->
+You can use the recorded admission requests to write your unit tests.
```

Comment:
I believe you can remove it. I guess the word "recorded" is used because is not unusual to actual record this data from a real Kubernetes request. Just to generate the test data. However, I think we can remove it because the files used in the test are a stripped version of the json sent from k8s.

Reply:
ok, thanks.